qiita_pet/handlers/api_proxy/artifact.py

```diff
@@ -53,3 +57,126 @@ def artifact_graph_get_req(artifact_id, direction, user_id):
             'node_labels': node_labels,
             'status': 'success',
             'message': ''}
+
+
+def artifact_get_req(artifact_id, user_id):
+    """Get information about the artifact
+
+    Parameters
+    ----------
+    artifact_id : int
+        Artifact being acted on
+    user_id : str
+        The user requesting the action
+
+    Returns
+    -------
+    dict
+        information about the artifact
+    """
+    pd = Artifact(int(artifact_id))
+    access_error = check_access(pd.study.id, user_id)
+    if access_error:
+        return access_error
+
+    try:
+        can_be_submitted_to_ebi = pd.can_be_submitted_to_ebi
+    except QiitaDBOperationNotPermittedError:
+        can_be_submitted_to_ebi = False
+
+    try:
+        can_be_submitted_to_vamps = pd.can_be_submitted_to_vamps
+    except QiitaDBOperationNotPermittedError:
+        can_be_submitted_to_vamps = False
+
+    try:
+        is_submitted_to_vamps = pd.is_submitted_to_vamps
+    except QiitaDBOperationNotPermittedError:
+        is_submitted_to_vamps = False
+
+    try:
```

Comment:
I guess this is only valid if can_be_submitted_to_ebi is True so to avoid the try/except you could simply test that first.